packages/flutter/lib/src/rendering/child_view.dart

```diff
@@ -32,6 +35,39 @@ mojom.ViewProxy _initViewProxy() {
 final mojom.ViewProxy _viewProxy = _initViewProxy();
 final mojom.View _view = _viewProxy?.ptr;
 
+mojom.ViewContainer _initViewContainer() {
+  mojom.ViewContainerProxy viewContainerProxy = new mojom.ViewContainerProxy.unbound();
+  _view.getContainer(viewContainerProxy);
+  viewContainerProxy.ptr.setListener(new mojom.ViewContainerListenerStub.unbound()..impl = _ViewContainerListenerImpl.instance);
+  return viewContainerProxy.ptr;
+}
+
+final mojom.ViewContainer _viewContainer = _initViewContainer();
+
+class _ViewContainerListenerImpl extends mojom.ViewContainerListener {
+  static final _ViewContainerListenerImpl instance = new _ViewContainerListenerImpl();
+
+  dynamic onChildAttached(int childKey, mojom.ViewInfo childViewInfo, [Function responseFactory = null]) {
+    try {
+      ChildViewConnection connection = _connections[childKey];
+      connection?._onAttachedToContainer(childViewInfo);
+    } finally {
+      return responseFactory();
```

Comment:
Yeah, I don't think it makes sense because responseFactory doesn't really have any important side effects.  We'll still fail to provide a response if we generate an exception.